spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe "Sign in flow" do
+  include TestFactories
+
+  describe "successfully" do
+    it "redirects to the topics index" do
+      user = authenticated_user
+      visit root_path
+      within '.user-info' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.